### ISSUE TITLE
Script styles and typefaces?

### ISSUE THREAD

**Issue by xfq** [w3c/clreq#275]
https://w3c.github.io/clreq/#four_commonly_used_typefaces_for_chinese_composition :

> There are four main typefaces in use for Chinese characters:

I think Song, Kai, Hei, and Fangsong are more like script styles than typefaces, because each of them contains many variations, and each of these variations is a typeface (see https://w3c.github.io/clreq/#fig-song for example).

**Comment by r12a** [comment]
fwiw, alreq these days refers to this kind of thing as 'writing styles' (see https://w3c.github.io/alreq/#h_writing_styles)

**Comment by ryukeikun** [comment]
correct, those should not be called as "typefaces".  "Writing styles" is good for me.

**Comment by c933103** [comment]
But wasn't Fangsong a specific type of type face following the characteristic of certain calligraphist from ancient time that follow the writing style of his time?

**Comment by xfq** [comment]
https://www.w3.org/International/articles/typography/fontstyles.en.html and https://www.w3.org/TR/typography/#fonts use the term "font styles".

**Comment by r12a** [comment]
Just a note on that.  Alreq continues to use the term 'writing styles', but it seemed to me when i wrote the article that 'font styles' felt more intuitive, so i used that term.  It does, of course, need to be distinguished from the CSS concept of 'font-style', but i thought it still worked OK.  I'm not aware of any industry terminology that is widespread.  Titus and i had a similar discussion before settling on 'writing style' for alreq (some years ago, now).

**Comment by c933103** [comment]
I think "Writing style" in arabic writing is more similar to the like of Big Seal Script in CJK from various nations in warring state period, instead of the different writing types of Fangsong and such.

**Comment by xfq** [comment]
There is a problem: a font is one of the implementations of a typeface, so using "font style" may cause ambiguity, because it confuses the concepts of different levels. "Types of typefaces" may be a better term.

**Comment by xfq** [comment]
@ryukeikun's suggestion: "typeface genre" or "typeface category".

**Comment by xfq** [comment]
@eisoch's suggestion: "We should have a W3C-wide, script-independent term for this concept, preferably defined in a W3C-wide glossary, and preferably avoid the word 'font' (for the same reason as https://github.com/w3c/clreq/issues/275#issuecomment-2809272520 )."

**Comment by r12a** [comment]
After trying various alternatives with the help of Co-Pilot AI, how about 'typeface style' ?

> Typeface style refers to the distinct visual characteristics of a typeface, including its weight, width, contrast, and decorative elements. It determines the overall aesthetic and readability of text. Typeface styles can range from serif, sans-serif, script, display, and monospaced, each serving different design and functional purposes

See also https://en.wikipedia.org/wiki/Typeface#Style_of_typefaces (there's a short subsection on Chinese)

**Comment by xfq** [comment]
May 19, 2025 Meeting: Eric was more inclined to "typeface genre", but he did not object to "typeface style", and neither did anyone else, so the final result was to use "typeface style".

**Comment by xfq** [comment]
> [@eisoch](https://github.com/eisoch)'s suggestion: "We should have a W3C-wide, script-independent term for this concept, preferably defined in a W3C-wide glossary, and preferably avoid the word 'font' (for the same reason as [#275 (comment)](https://github.com/w3c/clreq/issues/275#issuecomment-2809272520) )."

Working on this in https://github.com/w3c/i18n-actions/issues/174 .
